### ISSUE TITLE
Blue Feedback icon does not disappear after removing feedback 

### ISSUE THREAD

**Issue by Khalphion** [ls1intum/Artemis#693]
### Describe the bug

The new blue "!" exclamation mark does not disappear despite the feedback field being empty. 

#### To Reproduce

![image](https://user-images.githubusercontent.com/10117554/61952802-21a65100-afb5-11e9-99ad-085b04ee2c7c.png)

![image](https://user-images.githubusercontent.com/10117554/61952828-308d0380-afb5-11e9-8726-95f9fee20492.png)

![image](https://user-images.githubusercontent.com/10117554/61952837-38e53e80-afb5-11e9-8cbe-9cca7d6f47e6.png)

![image](https://user-images.githubusercontent.com/10117554/61952927-68944680-afb5-11e9-83dc-3cbf207dab05.png)

#### Expected behavior

The icon should be gone if no feedback is given on an element (behaviour before the update).

### Environment

<details><pre>
 - OS: macOS Mojave 10.14.3
 - Browser: Chrome 75.0.3770.142
 - Version: Artemis 3.4.1
</pre></details>

**Comment by krusche** [comment]
@TobiasPr maybe you can have a quick look, if this problem still occurs. If yes, this might be easy to fix.

**Comment by TobiasPr** [comment]
Yes, that is easily fixable. One question though: Whats the logic behind marking something with the blue exclamation mark? Is it just that there exists a written feedback?

Curernt logic in Apollon looks like this:
```              
{assessment.score === 0 && <FeedbackIcon />}
{assessment.score > 0 && <CorrectIcon />}
{assessment.score < 0 && <WrongIcon />}
```
so it does not take the feedback into account at all, its just evaluating the current score. Once set, the current score can never become null again, that's why we always have the exclamation mark

**Comment by TobiasPr** [comment]
fixed in apollon 2.0.11
